tests/utils/managed_deployment.py

```
@@ -0,0 +1,768 @@
+                    name=self._deployment_name,
+                )
+        except client.exceptions.ApiException as e:
+            if e.status != 404:  # Ignore if already deleted
+                raise
+
+    def port_forward(self, pod, remote_port, max_connection_attempts=3):
+        """Attempt to connect to a pod and return the port-forward object on success."""
+        port_forward = pod.portforward(
+            remote_port=remote_port,
+            local_port=0,
+            address="0.0.0.0",
```

Ruff (0.13.1)

Possible binding to all interfaces

(S104)